app/views/mailings/index.html.slim

```diff
@@ -1,24 +1 @@
-- if can? :crud, Mailing
-  nav.actions
-    ul.list-inline
-      li = link_to link_to icon('plus', 'New Mailing'), new_mailing_path, class: 'btn btn-sm btn-primary'
-
-h1 Mailings
-
-p.pagination-info
-  = page_entries_info @mailings
-
-- if @mailings.any?
-  table.table.table-striped.table-bordered.table-condensed
-    thead
-      tr
-        th Sent
-        th Subject
-        th Seasons
-        th Group
-        th To
-        th From
-    tbody
-      = render @mailings
-
-  = paginate @mailings, theme: 'twitter-bootstrap-3', pagination_class: 'pagination-sm'
+= render 'mailings/table', namespace: nil, mailings: @mailings
```

Comment:
Is the `table` partial supposed to be reused somewhere else or is this `index` view becoming more complex in the near future?  If not, I think it would make sense to the code from the partial into the actual view. Rendering things and looking up partials is probably one of the slowest parts of the framework, I don't see a point it utilizing it for no reason.

It also doesn't make things easier to grasp here in my opinion - correct me if I'm wrong.